specs/latest/1.0/index.html

```diff
@@ -877,6 +878,21 @@ <h4>Context creation parameters</h4>
               set to <code>false</code>, with the knowledge that a reduced-fidelity rendering mode
               should be used to improve performance.
             </dd>
+        <dt> <span class=prop-value>lowLatency</span></dt>
+            <dd>
+              <p>
+                If the value is true, the user agent will attempt to optimize the rendering of the
+                canvas to reduce latency, including potentially rendering from the front buffer.
+                This might desynchronize the canvas's display from the rest of the page and induce
+                tearing on some displays.
```

Comment:
How does this interact with preserveDrawingBuffer:false?

Reply:
It's orthogonal. lowLatency:true can be used with preserveDrawingBuffer set to either true or false.

In Chromium, if lowLatency is true and, currently, if experimental command-line flags are set, then (on some OSs) WebGL's drawing buffer is allocated in GPU memory that targets the scanout hardware directly. Then, if preserveDrawingBuffer is true, the WebGL-rendered canvas still maintains an internal buffer for rendering, and presentation involves one blit to the scanout buffer. If preserveDrawingBuffer is false, draw calls go directly to the scanout buffer. (Actually, I'm not 100% sure whether single-, double- or triple-buffers are maintained for those scanout buffers.)

One drawing app, currently the primary requestor of this API, has put it into practice using preserveDrawingBuffer:true. They do only incremental rendering upon receiving new input events. The lowLatency flag, in desynchronizing from the rest of the page and getting to the display hardware sooner, yields dramatic latency improvements.

We aim to continue to experiment on various platforms and refine the spec text here. However, the current text is sufficient to allow the desired optimizations.